packages/next-auth/src/jwt/index.ts

```diff
@@ -37,7 +37,7 @@ export async function decode(params: JWTDecodeParams): Promise<JWT | null> {
 
 export interface GetTokenParams<R extends boolean = false> {
   /** The request containing the JWT either in the cookies or in the `Authorization` header. */
-  req: NextApiRequest | Pick<NextApiRequest, "cookies" | "headers">
+  req: NextRequest | NextApiRequest | Pick<NextApiRequest, "cookies" | "headers">
```

Comment:
We'll need to import `NextRequest` from `next` above in order [for this to work](https://github.com/nextauthjs/next-auth/runs/6112902914?check_suite_focus=true):
```js
import type { NextApiRequest, NextRequest } from "next"
```

Reply:
Oh, yes... my bad. I updated it!